content/docs/analytics/Overview.md

```diff
@@ -37,6 +37,9 @@ With appbase.io analytics, you can now understand your search’s ROI impact thr
 - Popular Results
 - Distinct count of IP address/users
 - Popular countries
+- User sessions
+- User sessions' length
+- Bounce rate
 
 Here is a glimpse of what you can expect to measure with Appbase.io Analytics.
 ![Appbase.io analytics insights](https://i.imgur.com/4ZGfj5S.png)
```

Comment:
@bietkul Is this image updated with the session stats?

Reply:
@siddharthlatest I added a new image for `User Metrics` section, to update this image we need to populate some data which will require some time.